lib/appchain/transaction.rb

```diff
@@ -7,7 +8,7 @@ class VersionError < StandardError
 
     attr_accessor :to, :nonce, :quota, :valid_until_block, :data, :value, :chain_id, :version
 
-    # @param nonce [String]
+    # @param nonce [String] default is SecureRandom.hex, if you filled with nil or "", will set to random string
```

Comment:
```suggestion
    # @param nonce [String] default is SecureRandom.hex; if you provide with nil or empty string, it will be assigned a random string.
```